templates/_worker.js/index.ts

```diff
@@ -159,10 +159,21 @@ export default {
 						  pathname.replace(__BASE_PATH__, '/').replace('//', '/')
 						: pathname;
 
+					const nextPathnameMatcher = nextPathname.match(regexp);
+
 					if (
-						nextPathname.match(regexp) ||
+						nextPathnameMatcher ||
 						`${nextPathname}/page`.replace('//page', '/page').match(regexp)
 					) {
+						if (nextPathnameMatcher?.groups) {
+							const params = new URLSearchParams(nextPathnameMatcher.groups);
```

Comment:
you could just have
```ts
const params = Object.entries(nextPathnameMatcher.groups);
```
which to me looks much simpler/cleaner.

is using a `URLSearchParams` necessary for some reason?

Reply:
Doesn't appear to have any difference. Will use `Object.entries` instead.